src/components/AccountInfo.vue

```diff
@@ -0,0 +1,187 @@
+                    </path>
+                </clipPath>
+            </defs>
+        </svg>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Emit, Vue } from 'vue-property-decorator';
+import Identicon from './Identicon.vue';
+import Amount from './Amount.vue';
+import AddressDisplay from './AddressDisplay.vue';
+
+@Component({components: {Identicon, Amount, AddressDisplay}})
+export default class AccountInfo extends Vue {
+    @Prop(String) private address!: string;
+    @Prop(String) private shopLogoUrl?: string;
+    @Prop(String) private label?: string;
+    @Prop(String) private origin?: string;
+    @Prop(String) private walletLabel?: string;
+    @Prop(Number) private balance?: number;
+
+    private showShopLogo: boolean = true;
+
+    private get originDomain() {
+        console.log(this.origin);
+        return this.origin ? this.origin.split('://')[1] : '';
+    }
+
+    @Emit()
+    // tslint:disable-next-line no-empty
+    private close() {}
+}
+</script>
+
+<style scoped>
+    /* Nimiq Style */
+    .nq-label {
+        font-size: calc(1.75 * var(--nimiq-size, 8px));
+        line-height: 0.857;
+        font-weight: 600;
+        text-transform: uppercase;
+        letter-spacing: 0.107em;
+        margin: calc(1 * var(--nimiq-size, 8px)) 0;
+        opacity: 0.7;
+    }
+
+    .nq-icon {
+        width: 1em;
+        height: 1em;
+        background-repeat: no-repeat;
+        background-position: center;
+        display: inline-block;
+        background-size: 100%;
+        background-size: contain;
+    }
+
+    .nq-icon.cancel-circle {
+        background-image: url('data:image/svg+xml,<svg width="38" height="38" viewBox="0 0 38 38" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M9.80761 9.80762C14.8761 4.7391 23.1239 4.7391 28.1924 9.80762C33.2609 14.8761 33.2609 23.1239 28.1924 28.1924C23.1239 33.2609 14.8761 33.2609 9.80761 28.1924C4.73909 23.1239 4.73909 14.8761 9.80761 9.80762ZM19 17.5858L22.5355 14.0503L23.9497 15.4645L20.4142 19L23.9497 22.5355L22.5355 23.9498L19 20.4142L15.4645 23.9498L14.0503 22.5355L17.5858 19L14.0503 15.4645L15.4645 14.0503L19 17.5858Z" fill="%23101531"/></svg>');
+    }
+    /* END Nimiq Style */
+
+    .account-info {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        flex-grow: 1;
+        position: relative;
+        background: rgba(255, 255, 255, 0.95);
+        border-radius: calc(1 * var(--nimiq-size, 8px));
+        width: 100%;
+        height: 100%;
+        z-index: 29;
```

Comment:
1 z-index per millisecond of animation time. I think I stole this technique from Sebastian. It can probably be removed here, because the animating is not part of the component.

It needs to be at least `2`, however, to be above the header and footer gradients of the `AccountSelector`.

Reply:
I'm not a fan of that `z-index` transition technique.
The desired effect can usually be accomplished with `transition-delay` in a more elegant and more accurate way.
But yeah, here it's not used anyways.